src/renderer/components/+config-priority-classes/priority-classes-details.tsx

```diff
@@ -0,0 +1,54 @@
+/**
+ * Copyright (c) OpenLens Authors. All rights reserved.
+ * Licensed under MIT License. See LICENSE in root directory for more information.
+ */
+
+import "./priority-classes.scss";
+
+import React from "react";
+import { observer } from "mobx-react";
+import { DrawerItem } from "../drawer";
+import type { KubeObjectDetailsProps } from "../kube-object-details";
+import { PriorityClass } from "../../../common/k8s-api/endpoints";
+import { KubeObjectMeta } from "../kube-object-meta";
+import logger from "../../../common/logger";
+
+export interface PriorityClassesDetailsProps extends KubeObjectDetailsProps<PriorityClass> {
+}
+
+@observer
+export class PriorityClassesDetails extends React.Component<PriorityClassesDetailsProps> {
+
+  render() {
+    const { object: pc } = this.props;
+
+    if (!pc) {
+      return null;
+    }
+
+    if (!(pc instanceof PriorityClass)) {
+      logger.error("[PriorityClassesDetails]: passed object that is not an instanceof PriorityClass", pc);
+
+      return null;
+    }
```

Comment:
These defensive code shouldn't be required anymore. `enabled` in the `kubeObjectDetailItemInjectionToken` should take care of these.

Reply:
I removed this code.